pkg/addr/isdas.go

```diff
@@ -67,6 +70,8 @@ func MustParseISD(s string) ISD {
 	return isd
 }
 
+// @ requires isd >= 0
```

Comment:
given that an `isd` is a `uint16`, we know this property holds all the time (even though Gobra cannot prove it right now). As such, I would not put this in the specification, as we plan to fix this in Gobra. If you really need this fact, you should introduce an assumption instead

Reply:
Done.